api/controllers/todoListController.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+const Todo = mongoose.model('Todos')
+
+exports.list_all_todos = (req, res) => {
```

Comment:
list_all_todos => listAllTodos? I usually always use camelCase for js, since that's the standard in most repos